### ISSUE TITLE
render-engine init no commands static path

### ISSUE THREAD

**Issue by kjaymiller** [render-engine/render-engine#268]
![CleanShot 2023-10-04 at 11 22 49@2x](https://github.com/render-engine/render-engine/assets/8632637/e0b0fb00-e3d2-4921-8ee8-3f0119c0a7a6)

**Comment by tataraba** [comment]
The `create_app_py.txt` that is used by jinja to render the initial `app.py` file has this bit of code:

```
app = Site()
app.output_path = "{{output_path}}"
app.static_paths.add("{{static_paths}}")
```

This file is rendered from the initial CLI command (`init`)... since at that stage of the app lifecycle, a user would not have introduced a value for `static_paths` yet... so I don't think you need that third line. If you want to provide the ability to define `static_paths` as an option of the `init` command, then maybe you could do this:

```
{% if static_paths %}
app.static_paths.add("{{static_paths}}")
{% endif %}
```

However, it's much more likely that you just don't need the line at all.

The way the code is now, the line is introducing an empty string as the first element of `static_paths`. This messes up the theme manager for `_render_static`. As you start iterating through through `static_paths`, the first element is always empty because of that `app.static_paths.add("")` on the `app.py` file. This means that serving the sample app fails...

My suggestion is just to remove the line from `create_app_py.txt`.

Let me know if that works and I'll do the PR.

**Comment by kjaymiller** [comment]
The challenge with the init is balance. 

What is `render-engine init` designed for. While many will use it to build more and more apps quickly. I argue that it's a great tool for users building their first website with render-engine.

FOR THAT REASON: I think it is important to expose that value (and actually a few others that would be hard to find on your own without diving deep into the docs). In some prototypes, I've added (commented out) settings, plugins, and theme segments.

**Comment by tataraba** [comment]
Okay, I see what you mean. It is a little odd, because in essence, that line is adding an empty value, which you will have to deal with one way or another. If you leave as is, a user will get an error when they first try to run the app, so it would need to be dealt with elsewhere.

Maybe using the `_render_static` method in `themes.py` pop out any empty value?

**Comment by kjaymiller** [comment]
I agree that the existing value is incorrect but I don't think we remove the value completely I think you add the default so that folks are aware of the option

**Comment by kjaymiller** [comment]
fixed in #304
